Gopkg.toml

```diff
@@ -47,10 +47,10 @@ ignored = [
   name = "github.com/go-sql-driver/mysql"
   branch = "master"
 
-# https://github.com/grpc-ecosystem/grpc-gateway/commit/893772d
+# https://github.com/grpc-ecosystem/grpc-gateway/pull/497
 [[constraint]]
+  revision = "cde2f8f5bef06abdf9d0557f284f072f1cfa9f56"
   name = "github.com/grpc-ecosystem/grpc-gateway"
-  branch = "master"
```

Comment:
I understand if you consider this to be a break with your established methods, but I didn't want to go in and mess with the Gopkg.lock file itself to achieve the desired update.

Reply:
But why not leave this as is and run `dep ensure -update github.com/grpc-ecosystem/grpc-gateway`?

Reply:
I updated this explicitly because this was the revision your specified in [your comment](https://github.com/cockroachdb/cockroach/pull/20983#issuecomment-353478640), I misunderstood you if you meant I should just update to tip. I'll get that done.

Reply:
I think I need https://github.com/cockroachdb/vendored/tree/johanbrandhorst/bump-grpc-gateway updated.

Reply:
You might not. Let's see what CI says.